### PR TITLE
Update scripts to find the new location of dmd

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

**Comment by braddr** [comment]
See also closed issue 1 and 2.  Specifically, the windows build directory layout is not changed to match posix and this broke the testing on those platforms.  The changes had to be rolled back.  Get windows fixed please.

File: src/do_test_phobos.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ case "$2" in
         ;;
 esac
 
-$makecmd DMD=../dmd/src/dmd MODEL=$OUTPUT_MODEL $EXTRA_ARGS -f $makefile auto-tester-test >> ../phobos-unittest.log 2>&1
+$makecmd DMD=../dmd/generated/*/relese/$COMPILER_MODEL/dmd MODEL=$OUTPUT_MODEL $EXTRA_ARGS -f $makefile auto-tester-test >> ../phobos-unittest.log 2>&1
```
Comment (wilzbach) [inline src/do_test_phobos.sh]: release!

**Comment by RazvanN7** [comment]
@braddr could you tell us how to reproduce the error please? Is there a way to see the autotester?

**Comment by braddr** [comment]
I didn't leave the changes in the auto-tester to fail.  But you just need to build dmd on windows and observe that the output directory structure doesn't match the pattern.  A quick re-look, it's missing the release vs debug directory level.  It might well be incorrect in other ways/circumstances as well, but that stands out right now.

braddr@power:/home/braddr/sandbox/at-client (master)$ find master-test-Win_32_64 -name dmd.exe
master-test-Win_32_64/dmd/generated/Win_32_6432/dmd.exe
master-test-Win_32_64/dmd/src/dmd.exe

**Comment by andralex** [comment]
@RazvanN7 looks like you need to get a windows installation. Please coordinate with Eduard to get one going, you both need it. Thanks!

**Comment by RazvanN7** [comment]
ping @braddr I think we're good to go now

**Comment by braddr** [comment]
Both stable and master are updated?  Needs to be both.

**Comment by wilzbach** [comment]
> Both stable and master are updated? Needs to be both.

@braddr `master` has been [merged into stable 24 days ago (=2017-06-17))](https://github.com/dlang/dmd/pull/6915) - so I think all changes have been propagated to `stable` & we should be good to go here?

**Comment by wilzbach** [comment]
Ping @braddr - finally time to move forward with this?

**Comment by braddr** [comment]
This was completed quite a while ago, sorry for forgetting to close the issue.

**Pull request closed without merging**